lua/lua.go

```diff
@@ -611,7 +621,7 @@ func (L *State) StackTrace() []LuaStackEntry {
 func (L *State) RaiseError(msg string) {
 	st := L.StackTrace()
 	prefix := ""
-	if len(st) >= 1 {
+	if len(st) >= 2 {
 		prefix = fmt.Sprintf("%s:%d: ", st[1].ShortSource, st[1].CurrentLine)
```

Comment:
While implementing this Feature I noticed that RaiseError can actually index out of range.
Im not sure if the fix is checking length >= 2 or if the problem is that `st[1]...` should actually be `st[0]...`

Reply:
I think this is correct, probably.